docs/io/fits/index.rst

```diff
@@ -106,7 +106,7 @@ Working with large files
 The :func:`open` function supports a ``memmap=True`` argument that allows the
 array data of each HDU to be accessed with mmap, rather than being read into
 memory all at once.  This is particularly useful for working with very large
-arrays that cannot fit entirely into physical memory.
+arrays that cannot fit entirely into physical memory. Here ``memmap=True`` by defualt, and this value is obtained from a configuration item.
```

Comment:
typo (*defualt*). It should also mention the configuration item (`astropy.io.fits.Conf.use_memmap`).

Reply:
I've cancelled the CI given that there'll be a follow-up commit anyway.